anti-corruption-layer/src/main/java/com/iluwatar/corruption/system/modern/Shipment.java

```diff
@@ -0,0 +1,77 @@
+package com.iluwatar.corruption.system.modern;
+
+import java.util.Objects;
+
+/**
+ * The class represents a shipment in the modern system.
+ * The class is used by the modern system to store the data.
+ */
+public class Shipment {
```

Comment:
Use Lombok to get rid of boilerplate such as getters, setters, constructors, etc.